modules/gapi/src/api/render.cpp

```diff
@@ -1,15 +1,19 @@
+#include <stdexcept>
+
 #include <opencv2/imgproc.hpp>
 #include <opencv2/gapi/render/render.hpp>
 #include <opencv2/gapi/own/assert.hpp>
 
 #include "api/render_priv.hpp"
+#include "api/ocv_mask_creator.hpp"
+#include "api/ftp_render.hpp"
```

Comment:
It seems all header file changes above are not really needed here?

Reply:
Fixed